bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -47,6 +47,11 @@ struct ReplicaConfig {
   // numOfClientProxies >= 1
   uint16_t numOfClientProxies = 0;
 
+  // number of objects that represent external clients.
+  // numOfExternalClients >= 0
+  // By default numOfExternalClients will be 0 unless we configure it will be greater than.
```

Comment:
-> By default, numOfExternalClients is 0 unless configured differently.

Reply:
Fixed.